business/MetadataDatabase/Services/UserServices.cs

```diff
@@ -96,5 +103,65 @@ public IEnumerable<UserDto> FindByLogin(string login)
         {
             return this.userRepository.GetBySpecification(user => user.login == login).ToDto();
         }
+
+        /// <summary>
+        /// authenticate someone
+        /// </summary>
+        /// <param name="login">user login</param>
+        /// <param name="password">user password</param>
+        public UserDto Authenticate(string login, string password)
```

Comment:
Why not use Authenticate class as input ?

Reply:
yes good